www/index.php

```diff
@@ -23,21 +23,99 @@
 if (!$path)
 {
 	header('HTTP/1.1 500 Internal Server Error', null, 500);
+	header('Content-Type: text/html; charset=utf-8');
 	echo 'ERROR: Composer not properly set up! Run "composer install" or see README.md for more details' . PHP_EOL;
 
 	exit(1);
 }
 
 include $path;
 
+// Wrap in a try/catch so we can display an error if need be
+try
+{
+	$container = (new Joomla\DI\Container)
+		->registerServiceProvider(new JTracker\Service\ConfigurationProvider)
+		->registerServiceProvider(new JTracker\Service\DatabaseProvider)
+		->registerServiceProvider(new JTracker\Service\DebuggerProvider)
+		->registerServiceProvider(new JTracker\Service\DispatcherProvider)
+		->registerServiceProvider(new JTracker\Service\GitHubProvider)
+		->registerServiceProvider(new JTracker\Service\MonologProvider)
+		->registerServiceProvider(new JTracker\Service\TransifexProvider)
+		->registerServiceProvider(new JTracker\Service\WebApplicationProvider);
+
+	// Create the application aliases for the common 'app' key and base application class
+	$container->alias('Joomla\\Application\\AbstractApplication', 'JTracker\\Application')
+		->alias('app', 'JTracker\\Application');
+
+	// Create the logger aliases for the common 'monolog' key, the Monolog Logger class, and the PSR-3 interface
+	$container->alias('monolog', 'monolog.logger.application')
+		->alias('Monolog\\Logger', 'monolog.logger.application')
+		->alias('Psr\\Log\\LoggerInterface', 'monolog.logger.application');
+}
+catch (\Exception $e)
+{
+	if (isset($container))
+	{
+		// Try to write to a log
+		try
+		{
+			$logger = $container->get('monolog.logger.application');
+			$logger->critical(
+				sprintf(
+					'Exception of type %1$s thrown while booting the application',
+					get_class($e)
+				),
+				['exception' => $e]
+			);
+		}
+		catch (\Exception $nestedException)
+		{
+			// Do nothing, we tried our best
```

Comment:
Same argument as in the CLI front controller.  Can't say I'm too concerned about an error writing to the log file; the pertinent error message is displayed on screen.